db/fdb_fend.c

```diff
@@ -2633,6 +2643,12 @@ static int fdb_cursor_close(BtCursor *pCur)
               (fdbc->trans) ? fdbc->trans->seq : 0,
               fdbc->fcon.sock.sb);
 
```

Comment:
Does the 'dedup_cur' member need to be closed here via bdb_temp_table_close_cursor() ?

Reply:
`bdb_temp_table_close()` [closes open cursors](https://github.com/bloomberg/comdb2/blob/master/bdb/temptable.c#L1487-L1494). Therefore we need not to explicitly call  `bdb_temp_table_close_cursor()` here.

Reply:
Ok, in that case, everything looks good.